functions-and-scope.js

```diff
@@ -72,6 +141,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+const largestNumber = (number) => {
+    let highestGrade = 0;
+
+    for (let i = 0; i < grades.length; i++) {
```

Comment:
Volgens mij moest je hier de binnenkomende array gebruiken
```suggestion
    for (let i = 0; i < number.length; i++) {
```